my-travel-map/src/app/api/blog/post.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const sqlite3 = require('sqlite3');
+
+const app = express();
+app.use(express.json());
+
+// Provide the correct path to the existing myBlog.db
+const dbPath = '/Users/zhuminxuan/world-map-album-1/my-travel-map/myBlog.db'; // Replace with the actual path to your myBlog.db
```

Comment:
Besides, you should not create a new db file. You should only need to create a table.